src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/Messages.properties

```diff
@@ -141,6 +141,12 @@ ascanalpha.sourcecodedisclosure.gitbased.soln = Ensure that Git metadata files a
 ascanalpha.sourcecodedisclosure.gitbased.extrainfo =  
 ascanalpha.sourcecodedisclosure.gitbased.evidence = The source code for [{0}] was extracted using [{1}]
 
+ascanalpha.sqlinjection.mssql.name=SQL Injection - MsSQL
+ascanalpha.sqlinjection.mssql.desc=SQL injection may be possible
+ascanalpha.sqlinjection.mssql.soln=Do not trust client side input, even if there is client side validation in place. \nIn general, type check all data on the server side.\nIf the application uses JDBC, use PreparedStatement or CallableStatement, with parameters passed by '?'\nIf the application uses ASP, use ADO Command Objects with strong type checking and parameterized queries.\nIf database Stored Procedures can be used, use them.\nDo *not* concatenate strings into queries in the stored procedure, or use 'exec', 'exec immediate', or equivalent functionality!\nDo not create dynamic SQL queries using simple string concatenation.\nEscape all data received from the client.\nApply a 'whitelist' of allowed characters, or a 'blacklist' of disallowed characters in user input.\nApply the privilege of least privilege by using the least privileged database user possible.\nIn particular, avoid using the 'sa' or 'db-owner' database users. This does not eliminate SQL injection, but minimizes its impact.\nGrant the minimum database access that is necessary for the application.
+ascanalpha.sqlinjection.mssql.refs=https://www.owasp.org/index.php/Top_10_2010-A1\nhttps://www.owasp.org/index.php/SQL_Injection_Prevention_Cheat_Sheet
+ascanalpha.sqlinjection.mssql.alert.timebased.extrainfo=The query time is controllable using parameter value [{0}], which caused the request to take [{1}] milliseconds, parameter value [{2}], which caused the request to take [{3}] milliseconds, when the original unmodified query with value [{4}] took [{5}] milliseconds.
```

Comment:
The message does not seem to be right, I'm getting:
`The query time is controllable using parameter value [b WAITFOR DELAY '0:0:2' -- ], which caused the request to take [2,118] milliseconds, parameter value [b], which caused the request to take [0] milliseconds, when the original unmodified query with value [{4}] took [{5}] milliseconds.`

Should `when the original unmodified query with value [{4}] took [{5}] milliseconds` be removed? (or tweaked with previous sentence?)